subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/init/InitPluginIntegTest.groovy

```diff
@@ -0,0 +1,30 @@
+package org.shipkit.internal.gradle.init
+
+import org.gradle.testkit.runner.BuildResult
+import testutil.GradleSpecification
+
+class InitPluginIntegTest extends GradleSpecification {
+
+        def "runs initShipkit task in a project without any Shipkit configuration"() {
+            given:
+            projectDir.newFolder("gradle")
+
+            buildFile << """
+            apply plugin: "org.shipkit.java"
+        """
+
+            expect:
+            BuildResult result = pass("initShipkit", "-s")
+
+            result.tasks.collect { it.path }.join("\n") == """:identifyGitOrigin
```

Comment:
I would not assert on the tasks because it is enough to verify that the files were generated. Less code to read and maintain. However, I will leave it up to you :)